frontend/src/tests/components/eMIB/ResponseItem.test.js

```diff
@@ -0,0 +1,207 @@
+import React from "react";
+import { shallow } from "enzyme";
+import ResponseItem, { RESPONSE_TYPE } from "../../../components/eMIB/ResponseItem";
+import LOCALIZE from "../../../text_resources";
+
+it("renders the right response types (reply, reply all, forward)", () => {
+  //reply
+  const replyWrapper = shallow(
+    <ResponseItem
+      responseType={RESPONSE_TYPE.reply}
+      to={"to"}
+      cc={"cc"}
+      response={"response"}
+      reasonsForAction={"reasons"}
+    />
+  );
+
+  //reply all
+  const replyAllWrapper = shallow(
+    <ResponseItem
+      responseType={RESPONSE_TYPE.replyAll}
+      to={"to"}
+      cc={"cc"}
+      response={"response"}
+      reasonsForAction={"reasons"}
+    />
+  );
+
+  //forward
+  const forwardWrapper = shallow(
+    <ResponseItem
+      responseType={RESPONSE_TYPE.forward}
+      to={"to"}
+      cc={"cc"}
+      response={"response"}
+      reasonsForAction={"reasons"}
+    />
+  );
+
+  const reply = <span className="fas fa-reply" />;
```

Comment:
Use i tags instead of spans.

Reply:
Fixed in commit #https://github.com/code-for-canada/project-thundercat/pull/172/commits/2abecd409d82ffb6db5225b5de4b4a5c8a04aa09